libs/app-utils/models/src/lib/app-http.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { AxiosRequestConfig, AxiosResponse, Method } from "axios"
+
+export interface AppHttpRequestConfig extends AxiosRequestConfig {
+  url: string
+  method: Method
+  data?: Record<string, any>
+  params?: Record<string, any>
```

Comment:
Doesn't lint highlights this? Using any is most-likely an anti-pattern.

```suggestion
  data?: Record<string, unknown>
  params?: Record<string, unknown>
```